### DISCUSSION TITLE
[14 settembre 2023] Esercizio 1

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#257] [Esami]
Si progetti l’automa e la relativa rete sequenziale che riceve un input $\Large x$ e fornisce in output $\Large z$.

L’output $\Large z$ restituisce 1 se e solo se e solo se il numero naturale dato dagli ultimi 3 bit ricevuti, dà resto 1 quando diviso per 3.

Non sono ammesse sovrapposizioni.

Esempio

$$
    \Large INPUT: 11011000011110
$$

$$
    \Large \texttt{Output}: 00000010010010
$$

> [!NOTE]
> solo nella traccia del canale M-Z, per la rete sequenziale viene richiesto l'uso di porte logiche per la parte combinatoria e almeno FF JK.

<details closed>
    <summary> soluzione A-L </summary>

<div align="center">

![image](https://github.com/user-attachments/assets/fd12f6b2-9b13-4584-89db-8e4b149db0ef)
![image](https://github.com/user-attachments/assets/23b90340-913d-4fa6-8060-0c14ca688ce7)

</div>
</details>

**Comment by Nw4pS** [comment]
@FeddyLix17 perché in output c'è 1 alla sequenza 001? 1 non è divisibile per 3 con il resto di 1

**Reply by FeddyLix17** [reply]
1 / 3 fa 0 con resto di 1 🏃‍♂️‍➡️
